client/app/iplb/frontends/preview/iplb-frontends-preview.html

```diff
@@ -1,75 +1,75 @@
+        <dt class="preview-list__term"
+            data-translate="iplb_frontend_preview_dedicated_ip_failover"></dt>
+        <dd class="preview-list__definition">
+            <div class="oui-list" data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.dedicatedIpfo.length">
+                <ul class="oui-list__items">
+                    <li class="oui-list__item" data-ng-repeat="dedicatedIpfo in IpLoadBalancerFrontendPreviewCtrl.frontend.dedicatedIpfo track by $index"
+                        data-ng-bind="dedicatedIpfo"></li>
+                </ul>
+            </div>
+        </dd>
+        <dt class="preview-list__term"
+            data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.protocol !== 'udp'"
+            data-translate="iplb_frontend_preview_restrict_ip_access"></dt>
+        <dd class="preview-list__definition"
+            data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.protocol !== 'udp'"
+            data-ng-bind="IpLoadBalancerFrontendPreviewCtrl.frontend.allowedSource"></dd>
+        <dt class="preview-list__term"
+            data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.protocol === 'http'"
+            data-translate="iplb_frontend_preview_http_redirection"></dt>
+        <dd class="preview-list__definition"
+            data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.protocol === 'http'"
+            data-ng-bind="IpLoadBalancerFrontendPreviewCtrl.frontend.redirectLocation"></dd>
+        <dt class="preview-list__term"
+            data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.protocol === 'http'"
+            data-translate="iplb_frontend_preview_http_header"></dt>
+        <dd class="preview-list__definition"
+            data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.protocol === 'http'"
+            data-ng-bind="IpLoadBalancerFrontendPreviewCtrl.frontend.httpHeader"></dd>
+        <dt class="preview-list__term"
+            data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.protocol === 'http' && IpLoadBalancerFrontendPreviewCtrl.frontend.ssl"
+            data-translate="iplb_frontend_preview_hsts"></dt>
+        <dd class="preview-list__definition"
+            data-ng-show="IpLoadBalancerFrontendPreviewCtrl.frontend.protocol === 'http' && IpLoadBalancerFrontendPreviewCtrl.frontend.ssl"
+            data-ng-bind="IpLoadBalancerFrontendPreviewCtrl.frontend.hsts"></dd>
+        <dt class="preview-list__term"
+            data-translate="iplb_frontend_preview_active"></dt>
+        <dd class="preview-list__definition"
+            data-ng-bind="IpLoadBalancerFrontendPreviewCtrl.frontend.disabled ? 'false' : 'true'"></dd>
+        <dt class="preview-list__term"
+            data-translate="iplb_frontend_preview_id"></dt>
+        <dd class="preview-list__definition"
+            data-ng-bind="IpLoadBalancerFrontendPreviewCtrl.frontend.frontendId"></dd>
+    </dl>
+</oui-modal>
```

Comment:
Could you please replace the `data-ng-show` by `data-ng-if` and add `::` on `data-ng-bind` wherever it's possible